consensus/service/src/api/client_api_service.rs

```diff
@@ -138,13 +170,47 @@ impl ConsensusClientApi for ClientApiService {
             send_result(ctx, sink, result, logger)
         });
     }
+
+    fn propose_mint_config_tx(
+        &mut self,
+        ctx: RpcContext,
+        grpc_tx: mc_consensus_api::external::MintConfigTx,
+        sink: UnarySink<ProposeMintConfigTxResponse>,
+    ) {
+        let _timer = SVC_COUNTERS.req(&ctx);
+
+        if let Err(err) = self.authenticator.authenticate_rpc(&ctx) {
+            return send_result(ctx, sink, err.into(), &self.logger);
+        }
+
+        let mut result: Result<ProposeMintConfigTxResponse, RpcStatus> =
+            if counters::CUR_NUM_PENDING_VALUES.get() >= PENDING_LIMIT {
+                ConsensusGrpcError::OverCapacity.into()
+            } else if !(self.is_serving_fn)() {
+                ConsensusGrpcError::NotServing.into()
+            } else {
+                self.handle_propose_mint_config_tx(grpc_tx)
+                    .or_else(ConsensusGrpcError::into)
+            };
```

Comment:
That doesn't work:
```
error[E0308]: `if` and `else` have incompatible types
   --> consensus/service/src/api/client_api_service.rs:192:17
    |
189 |               } else if !(self.is_serving_fn)() {
    |  ____________________-
190 | |                 ConsensusGrpcError::NotServing
    | |                 ------------------------------ expected because of this
191 | |             } else {
192 | |                 self.handle_propose_mint_config_tx(grpc_tx)
    | |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected enum `ConsensusGrpcError`, found enum `Result`
193 | |             }
    | |_____________- `if` and `else` have incompatible types
```